app/assets/javascripts/components/common/wiki_select.jsx

```diff
@@ -107,7 +107,7 @@ const WikiSelect = createReactClass({
     const options = this.props.options;
     const filterOptions = function (val) {
       return options.filter(wiki =>
-        wiki.label.toLowerCase().includes(val.toLowerCase())
+        wiki.label.toLowerCase().startsWith(val.toLowerCase())
```

Comment:
One wrinkle I didn't consider previously: how will this work when you want to select "wikidata" as the wiki?